.github/workflows/package_for_release.yml

```diff
@@ -127,3 +132,36 @@ jobs:
               with:
                 name: live_compositor_with_web_renderer_darwin_aarch64.tar.gz
                 path: live_compositor_with_web_renderer_darwin_aarch64.tar.gz
+    docker:
+      runs-on: ubuntu-latest
+      steps:
+          - name: 📥 Checkout repo
+            uses: actions/checkout@v4
+
+          - name: 🔑 Log in to GitHub Container Registry
+            run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+          - name: 🔨 Build Live Compositor image
+            run: |
+              docker build -t ghcr.io/software-mansion/live-compositor:${{ inputs.version }} -f build_tools/docker/slim.Dockerfile .
+              docker image save ghcr.io/software-mansion/live-compositor:${{ inputs.version }} -o docker_image_live_compositor.tar
+              docker image rm ghcr.io/software-mansion/live-compositor:${{ inputs.version }}
+
+          - name: 📤 Upload Live Compositor artifact
+            uses: actions/upload-artifact@v4
+            with:
+              name: docker_image_live_compositor.tar
+              path: docker_image_live_compositor.tar
+
+          - name: 🔨 Build Live Compositor with Web Renderer image
+            run: |
+              rm docker_image_live_compositor.tar
+              docker build -t ghcr.io/software-mansion/live-compositor-with-web-renderer:${{ inputs.version }} -f build_tools/docker/full.Dockerfile .
```

Comment:
all images should be `ghcr.io/software-mansion/live-compositor` use tags to mark we renderer variant